lib/game_box_web/views/layout_view.ex

```diff
@@ -4,4 +4,36 @@ defmodule GameBoxWeb.LayoutView do
   # Phoenix LiveDashboard is available only in development by default,
   # so we instruct Elixir to not warn if the dashboard route is missing.
   @compile {:no_warn_undefined, {Routes, :live_dashboard_path, 2}}
+
+  def mount(_params, _session, socket) do
+    {:ok, socket}
+  end
+
+  attr :suffix, :string, default: nil
+
+  def navigation(assigns) do
+    ~H"""
+    <li>
+      <.link href={~p"/"} class="hover:!text-primary">
+        Home
+      </.link>
+    </li>
+    <%= if assigns[:current_user] do %>
```

Comment:
can we reference this with `@current_user`?

Reply:
Sure can!